tests/test_downloader_handlers.py

```diff
@@ -548,8 +548,8 @@ def test_download_with_content_length(self):
         # http://localhost:8998/partial set Content-Length to 1024, use download_maxsize= 1000 to avoid
         # download it
         yield crawler.crawl(seed=Request(url=self.mockserver.url('/partial'), meta={'download_maxsize': 1000}))
-        failure = crawler.spider.meta['failure']
-        self.assertIsInstance(failure.value, defer.CancelledError)
+        reason = crawler.spider.meta['close_reason']
+        self.assertTrue(reason, 'finished')
```

Comment:
Shouldn’t tests be failing now that you’ve added `raise defer.CancelledError` back?

Reply:
Oh, It's ```finished``` in both ways :) I'll return check on ```CancelledError```
thanks